src/expression.d

```diff
@@ -3695,6 +3697,13 @@ public:
         return resolve(loc ,sc, s, hasOverloads);
     }
 
+    /****************************************
+     * Resolve a symbol `s` to correcponding concrete Expression object.
```

Comment:
`corresponding`

I think a better description is that it resolves a symbol and wraps it in an expression object.